src/version-base.ts

```diff
@@ -278,7 +280,7 @@ export abstract class BaseVersioner {
         };
       })
       .sort((a, b) => {
-        return a.version.compare(b.version);
+        return b.version.compare(a.version);
```

Comment:
```suggestion
        // Sorted in descending order.
        return a.version.compare(b.version) * -1;
```

might be clearest haha